sdk/react/bottomOverlay.js

```diff
@@ -161,7 +161,7 @@ const BottomOverlay = createReactClass({
         testID={VIEW_NAMES.TIME_SEEK_BAR_THUMB}
         accessible={false}
         importantForAccessibility="no-hide-descendants"
-        accessibilityLabel={VIEW_NAMES.TIME_SEEK_BAR_THUMB}
+        accessibilityLabel=""
```

Comment:
Is it necessary to keep this prop with empty string?

Reply:
yes, without it all progress bar could not be focused in TalkBack mode